app/models/user_notification_preferences.rb

```diff
@@ -43,6 +43,10 @@ def update(enable_notifications:)
 
 private
 
+  def user_accredited_body_codes
+    user.providers.accredited_body.in_current_cycle.distinct.pluck(:provider_code)
```

Comment:
```suggestion
    user.providers.accredited_body.in_current_cycle.pluck(:provider_code)
```

https://github.com/DFE-Digital/teacher-training-api/blob/e45a0005585e96d53dd3de3c7c60f9ec1b0e2bb7/db/schema.rb#L243

Reply:
I think we still need the distinct as the user even though the provider record has that constraint, it's possible for the user to be associated with the same provider through more than one organisation so this association can return duplicates.

Reply:
fair call :+1: